Makefile

```diff
@@ -0,0 +1,75 @@
+# import config.
+# You can change the default config with `make cnf="config_special.env" build`
+cnf ?= container_config.env
+include $(cnf)
+export $(shell sed 's/=.*//' $(cnf))
+
+# HELP
+# This will output the help for each task
+.PHONY: help
+
+help: ## Display this help
+	@awk 'BEGIN {FS = ":.*?## "} /^[a-zA-Z_-]+:.*?## / {printf "\033[36m%-30s\033[0m %s\n", $$1, $$2}' $(MAKEFILE_LIST)
+
+.DEFAULT_GOAL := help
+
+CONTAINER_BUILD=
+CONTAINER_BUILD_OPT=
+CONTAINER_BUILD_ARG=
+CONTAINER_RUN=
+
+_set_no_cache:
+	$(eval CONTAINER_BUILD_OPT+=--no-cache)
+
+_set_docker_cmd:
+	$(eval CONTAINER_BUILD=docker build)
+	$(eval CONTAINER_RUN=docker run)
+
+_set_podman_cmd:
```

Comment:
What's podman? Do you use it often?

Reply:
Vtomole, Podman is container engine (https://podman.io/). Avaible from RedHat 8 distribution as primary container engine(https://www.redhat.com/en/blog/working-linux-containers-rhel-8-podman-image-builder-and-web-console). The cost to add the capability to run either podman &/or docker is small. The objectif is to provide both images format to gain an enlarged audience.